helm/aws-ebs-csi-driver-app/templates/crd-install/crd-job.yaml

```diff
@@ -26,6 +26,8 @@ spec:
       tolerations:
       - key: node-role.kubernetes.io/master
         effect: NoSchedule
+      - key: node-role.kubernetes.io/control-plane
+        effect: NoSchedule
```

Comment:
Now that I think of it, I propose to remove both `tolerations` from the job. Does anyone know why do we need the toleration in the first place? Why can't we just run the job in the workers?